<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>F.I.S.H</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="sideWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>375</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>375</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>375</height>
     </size>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="analyseButton">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>75</height>
         </size>
        </property>
        <property name="text">
         <string>Analyse</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="ajouterPoissonBoutton">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>75</height>
         </size>
        </property>
        <property name="text">
         <string>Ajouter un poisson</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="telechargerPoissonButton">
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>75</height>
         </size>
        </property>
        <property name="text">
         <string>Télécharger un poisson</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="mainWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>600</width>
      <height>375</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>375</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>375</height>
     </size>
    </property>
    <widget class="QWidget" name="analyseWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>375</height>
      </size>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>238</x>
        <y>272</y>
        <width>125</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Analyser</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>99</y>
        <width>571</width>
        <height>166</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="analyseLayout">
       <property name="leftMargin">
        <number>120</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="nomLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextBrowser" name="nomTextBrowser">
         <property name="maximumSize">
          <size>
           <width>201</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="dateLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lieuLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Lieu</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="imageLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Vidéo/Images</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTextBrowser" name="dateTextBrowser">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTextBrowser" name="lieuTextBrowser">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTextBrowser" name="upload">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="analyseParcourir">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Parcourir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="analyseLabel">
      <property name="geometry">
       <rect>
        <x>238</x>
        <y>30</y>
        <width>125</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Analyse</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ajoutPoissonWidget">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>375</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>561</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="ajoutPoissonLayout">
       <property name="leftMargin">
        <number>120</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="ajoutPoissonPoissonLabel">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Poisson</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ajoutePoissonLieuLabel">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Lieu</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ajoutPoissonImageLabel">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Image</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextBrowser" name="ajoutPoissonPoissonText">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextBrowser" name="ajoutPoissonLieuText">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTextBrowser" name="ajoutPoissonImageText">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="ajoutPoissonLabel">
      <property name="geometry">
       <rect>
        <x>212</x>
        <y>30</y>
        <width>175</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter un poisson</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajoutPoissonButton">
      <property name="geometry">
       <rect>
        <x>238</x>
        <y>272</y>
        <width>125</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Ajout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="telechargementPoissonWidget">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>375</height>
      </size>
     </property>
     <widget class="QWidget" name="formLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>561</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="ajoutPoissonLayout_2">
       <property name="leftMargin">
        <number>120</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="telechargerPoissonLieuLabel">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Poisson</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextBrowser" name="telechargerPoissonLieuText">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="ajoutPoissonParcourir">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Parcourir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="ajoutPoissonLabel_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>175</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ajouter un poisson</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>238</x>
        <y>272</y>
        <width>125</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Télécharger</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>mainWidget</zorder>
   <zorder>sideWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILES">
    <property name="title">
     <string>Fichier</string>
    </property>
   </widget>
   <addaction name="menuFILES"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
